samples/snippets/src/main/java/pubsublite/spark/AdminUtils.java

```diff
@@ -0,0 +1,198 @@
+import com.google.cloud.pubsublite.CloudRegion;
+import com.google.cloud.pubsublite.CloudZone;
+import com.google.cloud.pubsublite.ProjectId;
+import com.google.cloud.pubsublite.PublishMetadata;
+import com.google.cloud.pubsublite.SubscriptionName;
+import com.google.cloud.pubsublite.SubscriptionPath;
+import com.google.cloud.pubsublite.TopicName;
+import com.google.cloud.pubsublite.TopicPath;
+import com.google.cloud.pubsublite.cloudpubsub.Publisher;
+import com.google.cloud.pubsublite.cloudpubsub.PublisherSettings;
+import com.google.cloud.pubsublite.proto.Subscription;
+import com.google.cloud.pubsublite.proto.Topic;
+import com.google.protobuf.ByteString;
+import com.google.protobuf.util.Durations;
+import com.google.pubsub.v1.PubsubMessage;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+public class AdminUtils {
+
+  public static void createTopicExample(
+      String cloudRegion, char zoneId, String projectId, String topicId, int partitions)
+      throws Exception {
+
+    TopicPath topicPath =
+        TopicPath.newBuilder()
+            .setProject(ProjectId.of(projectId))
+            .setLocation(CloudZone.of(CloudRegion.of(cloudRegion), zoneId))
+            .setName(TopicName.of(topicId))
+            .build();
+
+    Topic topic =
+        Topic.newBuilder()
+            .setPartitionConfig(
+                Topic.PartitionConfig.newBuilder()
+                    // Set throughput capacity per partition in MiB/s.
+                    .setCapacity(
+                        Topic.PartitionConfig.Capacity.newBuilder()
+                            // Must be 4-16 MiB/s.
+                            .setPublishMibPerSec(4)
+                            // Must be 4-32 MiB/s.
+                            .setSubscribeMibPerSec(8)
+                            .build())
+                    .setCount(partitions))
+            .setRetentionConfig(
+                Topic.RetentionConfig.newBuilder()
+                    // How long messages are retained.
+                    .setPeriod(Durations.fromDays(1))
+                    // Set storage per partition to 30 GiB. This must be 30 GiB-10 TiB.
+                    // If the number of bytes stored in any of the topic's partitions grows
+                    // beyond this value, older messages will be dropped to make room for
+                    // newer ones, regardless of the value of `period`.
+                    .setPerPartitionBytes(30 * 1024 * 1024 * 1024L))
+            .setName(topicPath.toString())
+            .build();
+
+    AdminClientSettings adminClientSettings =
+        AdminClientSettings.newBuilder().setRegion(CloudRegion.of(cloudRegion)).build();
+
+    try (AdminClient adminClient = AdminClient.create(adminClientSettings)) {
+      Topic response = adminClient.createTopic(topic).get();
+      System.out.println(response.getAllFields() + "created successfully.");
+    }
```

Comment:
Can you catch and ignore `com.google.api.gax.rpc.AlreadyExistsException` errors here.